data/json/items/armor.json

```diff
@@ -10619,6 +10619,35 @@
     },
     {
         "type" : "ARMOR",
+        "id" : "legpouch",
+        "name" : "drop leg ammo pouch",
+        "name_plural" : "drop leg ammo pouches",
+        "weight" : 100,
+        "color" : "dark_gray",
+        "covers" : ["LEG_EITHER"],
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A small fabric ammo pouch that can be strapped to your leg and capable of holding a single magazine close at hand.",
+        "price" : 2000,
+        "material" : ["cotton", "null"],
+        "volume" : 1,
+        "warmth" : 0,
+        "phase" : "solid",
+        "encumbrance" : 3,
+        "flags" : ["VARSIZE", "WATER_FRIENDLY", "BELTED"],
+        "coverage" : 15,
+        "material_thickness" : 2,
+        "use_action" : {
+            "type": "holster",
+            "holster_prompt": "Stash ammo",
+            "holster_msg": "You stash your %s",
+            "max_volume": 2,
+            "draw_cost": 5,
+            "flags": [ "COMPACT_MAG" ]
```

Comment:
I saw the referenced note but I misunderstood it. Anyway, thanks for clarification.

>  Is there anywhere else?

No, I'm talking about exactly this place.